timescaledb/getting-started/compress-data.md

```diff
@@ -60,16 +48,33 @@ compression option and its `orderby` and `segmentby` attributes:
 SELECT * FROM timescaledb_information.compression_settings;
 ```
 
-Now that compression is enabled, schedule a policy to automatically compress
-data according to the settings defined above. Set a policy to compress data
-older than 10 years by using the following query:
+## Automatic compression
+
+Now that compression is enabled, you can schedule a [policy to automatically compress][compress-automatic]
+data according to the settings defined above. 
+
+Let's say that you want to compress data on your hypertable that has existed more than a month
+in the past. You could use the following code to do this:
 
 ```sql
--- Add compression policy
-SELECT add_compression_policy('weather_metrics', INTERVAL '10 years');
+SELECT add_compression_policy('stocks_real_time', INTERVAL '1 month');
 ```
 
-Just like for automated policies for continuous aggregates, we can view information and statistics about our compression background job in the following two information views:
+Similar to the continuous aggregates policy and retention policies, once you run this code, all data 
+from one month prior will be compressed in `stocks_real_time`, and a recurring compression
+policy is created. 
+
+<highlight type="note">
+You may be wondering, "Why not compress all your data? Why only compress data older than one month?".
+A compressed chunk cannot be updated and could block the addition or removal 
+of data. Since we will likely be adding data for more recent time periods, it is best  
+only to compress data you believe will not be updated. 
+ 
+  <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/continuous-aggregate-refresh-compression.jpg" alt="Continuous aggregate with refresh and compression policies"/>
+</highlight>
```

Comment:
```suggestion
It is important that you don't try to compress all your data. It is best to 
only compress data when it has aged, which is why the examples here 
compress data older than one month. You can't updated a compressed 
chunk, and this could block the addition or removal of data. Since you 
are likely to be adding data for more recent time periods, it is best only 
to compress data that is less likely to require updating. 

 <img class="main-content__illustration" src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/continuous-aggregate-refresh-compression.jpg" alt="Continuous aggregate with refresh and compression policies"/>
```

Few things here: I took it out of the admon, because having the image inside the admon makes it all a bit too big on the page, and took out the future tense ("will").  I also have a personal dislike for the question/answer format, so I reworded that while still trying to maintain the informal tone.